testsupport/space.go

```diff
@@ -84,16 +85,23 @@ func WithTierNameAndHashLabel(tierName, hash string) SpaceOption {
 	}
 }
 
+func WithCreationTimestamp(creationTimestamp time.Time) SpaceOption {
+	return func(s *toolchainv1alpha1.Space) {
+		s.CreationTimestamp = metav1.NewTime(creationTimestamp)
```

Comment:
Does it actually work? I mean does kube allow specifying the creation timestamp? Are you sure it's not ignored?

Reply:
right! locally was working but as you said the creationTimestamp it's being changed by the apiServer just before storing the object into etcd, so was just luck :D